libs/common/utilities/src/base/base.lit.element.ts

```diff
@@ -3,16 +3,19 @@ import { LitElement, PropertyDeclarations } from 'lit';
 export abstract class BaseLitElement extends LitElement {
     protected allowCustomCSS = true;
     private customCSS: string | null = null;
+    private customCSSPrefix: string | null = null;
 
     static get properties(): PropertyDeclarations {
         return {
-            customCSS: { type: String, attribute: 'data-vl-custom-css', reflect: true },
+            customCSS: { type: String, attribute: 'custom-css', reflect: true },
+            customCSSPrefix: { type: String, attribute: 'data-vl-custom-css', reflect: true },
         };
     }
 
     connectedCallback(): void {
         super.connectedCallback();
 
+        this.customCSS = this.customCSSPrefix ? this.customCSSPrefix : this.customCSS;
```

Comment:
Kan je dit in de addCustomCSS methode doen? Dan vervuil je de connectedCallback niet.

Reply:
done!